<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reassemble networks</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelOutFile">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Output file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonZoom">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>451</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCmgui">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Create CMGUI file</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_axes">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCheckBox" name="checkBox_x">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>X axis</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_y">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Y axis</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_z">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Z axis</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_axes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Axes cut</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_basename">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Block base name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBasename">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOutFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>outFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonZoom</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>am_joiner()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonClickHandler()</slot>
  <slot>outFileSelecter()</slot>
  <slot>am_joiner()</slot>
 </slots>
</ui>
